source/guide_goaccess.rst

```diff
@@ -0,0 +1,199 @@
+ [isabell@stardust ~]$
+
+Download the actual source code from GitHub. 
+
+::
+
+ [isabell@stardust ~]$ git clone https://github.com/allinurl/goaccess.git
+ [isabell@stardust ~]$
+
+Step 2 - Source Code Configuration, Compiling and Installation
+--------------------------------------------------------------
+
+Prepare the compile environment.
+
+::
+
+ [isabell@stardust ~]$ cd ~/filebase/goaccess/
+ [isabell@stardust ~]$ autoreconf -fiv
+ [isabell@stardust ~]$
+
+Configure and compile the source code with the commands ``configure``, ``make`` and finally install it with ``make install``.
+
+.. note:: Please use single steps instead of combining all three in one process to see and identify possible errors.
+
+Before we start, we have to consider some aspects, especially a shared hosting environment like Uberspace:
+
+ * ``--prefix=/home/isabell``: Installation target for your personal Uberspace.
+ * ``--enable-utf8``: Compile with wide character support.
+ * ``--enable-geoip=legacy``: Compile with GeoLocation support, legacy will utilize the original GeoIP databases.
+
+Other options can be found in the GoAccess `installation documentation`_.
+
+.. warning:: Replace ``isabell`` with your Uberspace name!
+
+::
+
+ [isabell@stardust ~]$ cd ~/filebase/goaccess/
+ [isabell@stardust ~]$ ./configure --enable-utf8 --enable-geoip=legacy --prefix=/home/isabell
+ [isabell@stardust ~]$ make
+ [isabell@stardust ~]$ make install
+ [isabell@stardust ~]$
+
+Configuration
+=============
+
+After the installation of GoAccess, it is necessary to enable the web server logs and to configure GoAccess with the right log format.
+
+Step 1 - Enable the Web Server Log
+----------------------------------
+
+Please follow the instructions by the Uberspace manual_ to enable the web server logs.
+
+.. note:: Please consider, after the web server log enabling, it needs some time to have some entries in your new log files. Depends on the web traffic.
+
+Step 2 - GoAccess Configuration
+-------------------------------
+
+Edit the configuration file ``~/etc/goaccess/goaccess.conf`` and reach out for the following parameters to uncomment these.
+
+.. code-block:: bash
+ :emphasize-lines: 2,4,6,8
+ 
+ # Time Format Options (required)
+ time-format %H:%M:%S
+ 
+ # Date Format Options (required)
+ date-format %d/%b/%Y
+ 
+ # NCSA Combined Log Format (is in use by Uberspace)
+ log-format %h %^[%d:%t %^] "%r" %s %b "%R" "%u"
+ 
+ # Set HTML report page title and header.
+ html-report-title My Uberspace
+
+Launch
+======
+
+Step 1 - First Try (or Realtime Analysis in the Shell)
+------------------------------------------------------
+
+To get first results, to check that everthing is maintained, please enter:
+
+::
+
+ [isabell@stardust ~]$ goaccess -a -p ~/etc/goaccess/goaccess.conf -f ~/logs/webserver/access_log
+
+Scroll with your cursor keys up and down. With "q" you can quit GoAccess.
+
+The used parameters are:
+
+ * ``-a``: Enable a list of user-agents by hosts.
```

Comment:
What do you think about using the long-options `--agent-list`, `--config-file=` and `--log-file=` instead to make the command self-explanatory, instead of explaining it below?